crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -79,7 +80,15 @@ impl JsonRpcHandler {
         _block_id: BlockId,
         _contract_address: ContractAddressHex,
     ) -> RpcResult<ClassHashHex> {
-        Err(error::ApiError::ContractNotFound)
+        let parsed_address = _contract_address.0.try_into().unwrap(); // TODO better error handling
```

Comment:
```suggestion
        let parsed_address = _contract_address.0.try_into()?;
```

Reply:
Wasn't as simple as that because I had to map the error. Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/50/commits/7e468c61b374117e62f877894b80f76ae2340c77